docs/en/Configuration-Properties.md

```diff
@@ -23,7 +23,7 @@ The common configuration contains constants shared by different components.
 <tr><th>Property Name</th><th>Default</th><th>Description</th></tr>
 {% for item in site.data.table.common-configuration %}
   <tr>
-    <td>{{ item.propertyName }}</td>
+    <td><a class="anchor" name="{{ item.propertyName }}"></a> {{ item.propertyName }}</td>
```

Comment:
I don't know much about this, but is this an empty `<a>` element and what does that mean? What would happen if you added the class and anchor to the `<tr>` row?